<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>setting</class>
 <widget class="QWidget" name="setting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>732</width>
    <height>580</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Setting</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_4">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3" stretch="4,1,2">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="cursor">
        <cursorShape>ArrowCursor</cursorShape>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="title">
        <string notr="true">Weights</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>20</y>
          <width>621</width>
          <height>271</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,4">
         <property name="sizeConstraint">
          <enum>QLayout::SetMinAndMaxSize</enum>
         </property>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,0">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>content weight</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSlider" name="horizontalSlider">
               <property name="maximum">
                <number>20000</number>
               </property>
               <property name="value">
                <number>1</number>
               </property>
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDoubleSpinBox" name="doubleSpinBox">
               <property name="decimals">
                <number>0</number>
               </property>
               <property name="maximum">
                <double>20000.000000000000000</double>
               </property>
               <property name="value">
                <double>1.000000000000000</double>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QLabel" name="label">
               <property name="text">
                <string>style weight</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSlider" name="horizontalSlider_3">
               <property name="minimum">
                <number>1</number>
               </property>
               <property name="maximum">
                <number>20000</number>
               </property>
               <property name="value">
                <number>10000</number>
               </property>
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDoubleSpinBox" name="doubleSpinBox_3">
               <property name="decimals">
                <number>0</number>
               </property>
               <property name="minimum">
                <double>1.000000000000000</double>
               </property>
               <property name="maximum">
                <double>20000.000000000000000</double>
               </property>
               <property name="singleStep">
                <double>1.000000000000000</double>
               </property>
               <property name="value">
                <double>10000.000000000000000</double>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout" stretch="0,1,1">
           <property name="spacing">
            <number>3</number>
           </property>
           <property name="sizeConstraint">
            <enum>QLayout::SetDefaultConstraint</enum>
           </property>
           <item>
            <widget class="QTreeWidget" name="treeWidget_2">
             <property name="maximumSize">
              <size>
               <width>619</width>
               <height>1000</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="uniformRowHeights">
              <bool>false</bool>
             </property>
             <column>
              <property name="text">
               <string notr="true">imgs</string>
              </property>
              <property name="font">
               <font>
                <pointsize>9</pointsize>
               </font>
              </property>
             </column>
             <column>
              <property name="text">
               <string>weight</string>
              </property>
             </column>
             <item>
              <property name="text">
               <string>set imgs weight</string>
              </property>
              <property name="font">
               <font>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="checkState">
               <enum>Checked</enum>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <widget class="QTableWidget" name="tableWidget">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="maximumSize">
              <size>
               <width>619</width>
               <height>57</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="verticalScrollBarPolicy">
              <enum>Qt::ScrollBarAsNeeded</enum>
             </property>
             <property name="horizontalScrollBarPolicy">
              <enum>Qt::ScrollBarAsNeeded</enum>
             </property>
             <property name="sizeAdjustPolicy">
              <enum>QAbstractScrollArea::AdjustToContents</enum>
             </property>
             <property name="verticalScrollMode">
              <enum>QAbstractItemView::ScrollPerItem</enum>
             </property>
             <property name="showGrid">
              <bool>true</bool>
             </property>
             <property name="gridStyle">
              <enum>Qt::SolidLine</enum>
             </property>
             <property name="sortingEnabled">
              <bool>false</bool>
             </property>
             <attribute name="horizontalHeaderVisible">
              <bool>true</bool>
             </attribute>
             <attribute name="horizontalHeaderCascadingSectionResizes">
              <bool>false</bool>
             </attribute>
             <attribute name="horizontalHeaderHighlightSections">
              <bool>true</bool>
             </attribute>
             <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
              <bool>false</bool>
             </attribute>
             <attribute name="horizontalHeaderStretchLastSection">
              <bool>false</bool>
             </attribute>
             <attribute name="verticalHeaderVisible">
              <bool>true</bool>
             </attribute>
             <attribute name="verticalHeaderCascadingSectionResizes">
              <bool>false</bool>
             </attribute>
             <attribute name="verticalHeaderHighlightSections">
              <bool>true</bool>
             </attribute>
             <attribute name="verticalHeaderShowSortIndicator" stdset="0">
              <bool>false</bool>
             </attribute>
             <attribute name="verticalHeaderStretchLastSection">
              <bool>false</bool>
             </attribute>
             <row>
              <property name="text">
               <string>style layer weight</string>
              </property>
             </row>
             <column>
              <property name="text">
               <string>relu1_1</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>relu2_1</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>relu3_1</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>relu4_1</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>relu5_1</string>
              </property>
             </column>
             <item row="0" column="0">
              <property name="text">
               <string>0.2</string>
              </property>
              <property name="textAlignment">
               <set>AlignCenter</set>
              </property>
             </item>
             <item row="0" column="1">
              <property name="text">
               <string>0.2</string>
              </property>
              <property name="textAlignment">
               <set>AlignCenter</set>
              </property>
             </item>
             <item row="0" column="2">
              <property name="text">
               <string>0.2</string>
              </property>
              <property name="textAlignment">
               <set>AlignCenter</set>
              </property>
             </item>
             <item row="0" column="3">
              <property name="text">
               <string>0.2</string>
              </property>
              <property name="textAlignment">
               <set>AlignCenter</set>
              </property>
             </item>
             <item row="0" column="4">
              <property name="text">
               <string>0.2</string>
              </property>
              <property name="textAlignment">
               <set>AlignCenter</set>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <widget class="QTableWidget" name="tableWidget_3">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="maximumSize">
              <size>
               <width>237</width>
               <height>57</height>
              </size>
             </property>
             <property name="verticalScrollBarPolicy">
              <enum>Qt::ScrollBarAlwaysOff</enum>
             </property>
             <property name="horizontalScrollBarPolicy">
              <enum>Qt::ScrollBarAlwaysOff</enum>
             </property>
             <property name="showGrid">
              <bool>true</bool>
             </property>
             <property name="sortingEnabled">
              <bool>false</bool>
             </property>
             <property name="wordWrap">
              <bool>true</bool>
             </property>
             <property name="cornerButtonEnabled">
              <bool>false</bool>
             </property>
             <row>
              <property name="text">
               <string>content layer weight</string>
              </property>
             </row>
             <column>
              <property name="text">
               <string>conv4_2</string>
              </property>
             </column>
             <item row="0" column="0">
              <property name="text">
               <string>1</string>
              </property>
              <property name="font">
               <font>
                <family>Microsoft YaHei</family>
                <stylestrategy>NoAntialias</stylestrategy>
               </font>
              </property>
              <property name="textAlignment">
               <set>AlignCenter</set>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="title">
        <string>Color</string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>30</y>
          <width>101</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>orininal colors:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_10">
        <property name="geometry">
         <rect>
          <x>410</x>
          <y>30</y>
          <width>121</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>color convert type:</string>
        </property>
       </widget>
       <widget class="QComboBox" name="comboBox_4">
        <property name="geometry">
         <rect>
          <x>540</x>
          <y>30</y>
          <width>81</width>
          <height>22</height>
         </rect>
        </property>
        <item>
         <property name="text">
          <string>YUV</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>YCrCb</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>CIE L*a*b*</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>CIE L*u*v* </string>
         </property>
        </item>
       </widget>
       <widget class="QRadioButton" name="radioButton">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>30</y>
          <width>201</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Transfer the style and colors</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
        <property name="autoRepeat">
         <bool>false</bool>
        </property>
       </widget>
       <widget class="QRadioButton" name="radioButton_2">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>50</y>
          <width>251</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Transfer the style but not the colors</string>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="title">
        <string>Solve</string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>20</y>
          <width>81</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>pooling type:</string>
        </property>
        <property name="textFormat">
         <enum>Qt::AutoText</enum>
        </property>
        <property name="scaledContents">
         <bool>false</bool>
        </property>
       </widget>
       <widget class="QComboBox" name="comboBox">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>20</y>
          <width>121</width>
          <height>22</height>
         </rect>
        </property>
        <item>
         <property name="text">
          <string>average pooling</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>max pooling</string>
         </property>
        </item>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>60</y>
          <width>61</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>optimizer:</string>
        </property>
       </widget>
       <widget class="QComboBox" name="comboBox_2">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>60</y>
          <width>69</width>
          <height>22</height>
         </rect>
        </property>
        <item>
         <property name="text">
          <string>L-BFGS</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Adam</string>
         </property>
        </item>
       </widget>
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>360</x>
          <y>20</y>
          <width>91</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>learning rate:</string>
        </property>
       </widget>
       <widget class="QDoubleSpinBox" name="doubleSpinBox_2">
        <property name="geometry">
         <rect>
          <x>470</x>
          <y>20</y>
          <width>62</width>
          <height>22</height>
         </rect>
        </property>
        <property name="minimum">
         <double>0.010000000000000</double>
        </property>
        <property name="maximum">
         <double>10.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.010000000000000</double>
        </property>
        <property name="value">
         <double>1.000000000000000</double>
        </property>
       </widget>
       <widget class="QLabel" name="label_8">
        <property name="geometry">
         <rect>
          <x>360</x>
          <y>50</y>
          <width>101</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>max iterations:</string>
        </property>
       </widget>
       <widget class="QSpinBox" name="spinBox">
        <property name="geometry">
         <rect>
          <x>470</x>
          <y>50</y>
          <width>51</width>
          <height>22</height>
         </rect>
        </property>
        <property name="maximum">
         <number>2000</number>
        </property>
        <property name="value">
         <number>1000</number>
        </property>
       </widget>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>90</y>
          <width>131</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>content loss function:</string>
        </property>
       </widget>
       <widget class="QComboBox" name="comboBox_3">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>90</y>
          <width>181</width>
          <height>22</height>
         </rect>
        </property>
        <item>
         <property name="text">
          <string>type1: 1/(2*N**0.5*M**0.5)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>type2: 1/(N*M)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>type3: 1/2</string>
         </property>
        </item>
       </widget>
       <widget class="QLabel" name="label_9">
        <property name="geometry">
         <rect>
          <x>360</x>
          <y>90</y>
          <width>54</width>
          <height>12</height>
         </rect>
        </property>
        <property name="text">
         <string>device:</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="radioButton_3">
        <property name="geometry">
         <rect>
          <x>420</x>
          <y>90</y>
          <width>89</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>GPU</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="radioButton_4">
        <property name="geometry">
         <rect>
          <x>470</x>
          <y>90</y>
          <width>89</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>CPU</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QLabel" name="label_11">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>130</y>
          <width>251</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>max size of width or height of picture:</string>
        </property>
       </widget>
       <widget class="QSpinBox" name="spinBox_2">
        <property name="geometry">
         <rect>
          <x>270</x>
          <y>130</y>
          <width>51</width>
          <height>22</height>
         </rect>
        </property>
        <property name="minimum">
         <number>512</number>
        </property>
        <property name="maximum">
         <number>2048</number>
        </property>
        <property name="value">
         <number>1024</number>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
